frontend/app/components/ToolCard.tsx

```diff
@@ -128,6 +134,13 @@ export default function ToolCard (props: ToolCardProps) {
                 />
               )
           }
+          {
+            tool.navigation_enabled && (
```

Comment:
Is it possible to add the tooltip to the "Add Tool" and "Remove Tool" button, so that maybe when people mouse over to those action buttons, they can also see the message?

Reply:
This is something we could consider; I'm not sure yet of the benefit. Are you thinking that the "REMOVE TOOL" button would have the same tool tip the icon, or that the buttons would have different tool tips? I guess I'd be concerned with the UI getting cluttered, or repeating things unnecessarily. We could ask the UX team.

Reply:
I was thinking within the , there are both and elements, so that same info text can be shown when user hover over the "Remove Tool" button, too.

Same setting likewise for the "Add Tool" button.

But I am OK with hearing from the UX team first.

Reply:
Still not exactly clear. When you hover over or have in focus the add or remove button, where would the tooltip be and what would it say? On the button or on the icon?

Reply:
Sam and I had a discussion afterwards. We agree to wait for feedback from UX team first. Resolve this for now.